roles/clevis/tasks/create-tangcrypt.yml

```diff
@@ -0,0 +1,58 @@
+---
+
+- name: Remove dropbear ssh initramfs packages
+  become: true
+  ansible.builtin.apt:
+    name: "{{ dropbear_packages }}"
+    state: absent
+  notify:
+    - Update initramfs
+
+- name: Remove dropbear artefacts
+  ansible.builtin.file:
+    dest: /etc/initramfs-tools/conf.d/dropbear.conf
+    state: absent
+
+- name: "Check if disk {{ clevis_root_disk }} already bind to tang"
+  become: true
+  ansible.builtin.command: "clevis luks list -d {{ clevis_root_disk }} -s 2"
+  register: result_clevis_luks_list
+  ignore_errors: true
+  changed_when: false
+
+- name: Prepare clevis environment
+  ansible.builtin.tempfile:
+    state: directory
+    suffix: clevis
+  register: tmp_clevis_env
+
+- name: Get adv handshake from tang server
+  become: true
+  ansible.builtin.uri:
+    url: "{{ clevis_tang_url }}/adv"
+    dest: "{{ tmp_clevis_env.path }}/adv.jws"
+    method: GET
+    group: root
+    owner: root
+    mode: 0400
+  changed_when: "result_clevis_luks_list.rc == 1"
+
+- name: Insert the keyfile for luks unlock
+  become: true
+  ansible.builtin.copy:
+    content: "{{ clevis_initial_luks_password }}"
+    dest: "{{ tmp_clevis_env.path }}/pass"
+    group: root
+    owner: root
+    mode: 0400
+  changed_when: "result_clevis_luks_list.rc == 1"
+
+- name: Create the tang pin
+  become: true
+  ansible.builtin.command: "/bin/true"
```

Comment:
This looks even weirder, why do you need to run this? Can't you attach the notify to the previous command?

Reply:
correct with  the previous command